README.md

```diff
@@ -65,8 +64,8 @@ actionView.setListener(new SlideActionView.SlideActionListener() {
     // slid right
   }
 });
-```
+ ``` 
```

Comment:
Clearly mention the part in future-work as discussed.

Reply:
Done